packages/tailwindcss-x/tsconfig.eslint.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "noEmit": true
```

Comment:
We already have this set on the parent tsconfig.json, so we should skip it here.

Reply:
actually, I think we are going to export types from this package so we have to remove that option from the parent.